### ISSUE TITLE
SNS

### ISSUE THREAD

**Issue by teddywilson** [bascan/aws-dropwizard#12]
I am interested in building a similar library that offers support for SNS (Simple Notification Service). I was wondering if you would be interested in me integrating my changes here and making this a library that supports both SQS and SNS, or if you would be okay with me refactoring some of your code and making a separate project exclusively for SNS. I figured I would reach out before starting anything!

@bascan

**Comment by bascan** [comment]
Hi Theodore,

Thanks for reaching out!
Integrating your changes and adding support for SNS to this library sounds
like a great idea.

I can give you full access to the repo and you're good to go. I can release
your changes to Maven Central when you're ready.

Best regards,
Bas

**Comment by teddywilson** [comment]
Hey @bascan,

That sounds great! I tried cloning it last night and had some issues with Gradle, but I'm going to give it another try later this evening. I will be in touch soon.

**Comment by bascan** [comment]
Hey @teddywilson 

Many thanks for your PR! It's reviewed and merged into master. I invited you for push access to speed things up. I also renamed the project to aws-dropwizard as requested. I'll change the Maven artifact name upon release, so that it matches the new project name.

Please let me know when you feel the codebase is ready for release to Maven Central.

Cheers,
Bas

**Comment by teddywilson** [comment]
Hey @bascan 

I believe these changes are ready to be rolled out -- thanks for letting me contribute!
